lib/compress/zstd_compress.c

```diff
@@ -3525,17 +3755,16 @@ ZSTD_CDict* ZSTD_initStaticCDict(void* workspace, size_t workspaceSize,
     }
     cdict->refContext = ZSTD_initStaticCCtx(ptr, cctxSize);
 
-    if (ZSTD_isError( ZSTD_initCDict_internal(cdict,
-                                              dict, dictSize,
-                                              1 /* byReference */, dictMode,
-                                              cParams) ))
+    if (ZSTD_isError( ZSTD_initCDict_internal(cdict, dict, dictSize,
```

Comment:
This diff is already pretty large, can we avoid all of the formatting-only changes?